docs/en/integrations/index.mdx

```diff
@@ -129,6 +130,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Mode Analytics|<img src={require('./images/logos/logo_mode.png').default} class="image" alt="Mode logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data visualization|Business Intelligence built around data teams|[Website](https://mode.com/)|
 |Openblocks|<img src={require('./images/logos/logo_openBlocks.png').default} class="image" alt="Openblocks logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|SQL client|Openblocks is a low code platform for building UIs|[Documentation](https://docs.openblocks.dev/data-sources/connect-to-databases/clickhouse/)|
 |Prequel|<PrequelSVG style={{width: '3rem'}} />|Data sharing|Connect your ClickHouse instance to Prequel to share data to or sync data from your users and partners.|[Documentation](https://docs.prequel.co/docs/sources-clickhouse-generic)|
+|RisingWave|<RisingWaveSVG style={{width: '3rem'}} />|Data ingestion| SQL stream processing with a Postgres-like experience. 10x faster and more cost-efficient than Apache Flink. |[Documentation](https://www.risingwave.dev/docs/current/intro/)|
```

Comment:
Could you attach a link to ClickHouse integration? Also the docs says ClickHouse support is not planned.
![2023-07-06_10-43-42](https://github.com/ClickHouse/clickhouse-docs/assets/3198181/09c5a6a5-2db9-4b29-b7c0-4aa606d1af5f)

Reply:
@hengm3467 when https://www.risingwave.dev/docs/upcoming/sink-to-clickhouse/ will be available on the website?

Reply:
@mshustov Thanks for checking! It's already available / published on the website. It's just we haven't released the official version that includes this support. And I'm not sure if the link for the dev version (the one you mentioned above) is good enough, or we should wait for the official version to be ready and use the link for the official version (https://www.risingwave.dev/docs/current/sink-to-clickhouse/ ) instead? Appreciate any suggestions you may have. 

Thanks.

Reply:
> r we should wait for the official version to be ready and use the link for the official version (https://www.risingwave.dev/docs/current/sink-to-clickhouse/ ) instead

Let's wait for the official docs to be updated. Otherwise, it will be confusing for the customers that land on the website from internet

Reply:
@mshustov I have just updated the link to the RisingWave-ClickHouse integration page. Please review. Thanks.